wrappers/vroom.rb

```diff
@@ -384,7 +388,7 @@ def run_vroom(vrp, dimensions, _job)
       output = Tempfile.new('optimize-vroom-output', @tmp_dir)
       output.close
 
-      cmd = "#{@exec_vroom} -i '#{input.path}' -o '#{output.path}'"
+      cmd = "#{@exec_vroom} -i '#{input.path}' -o '#{output.path}' -x 0"
```

Comment:
Can you add a `TODO:` and share the link of this PR in the ticket? It looks like we will move the ticket to the next milestone and make a full performance test..

Reply:
I didn't check but I suspect Flore's instance has no TWs which renders "everything possible" which degrades the performance of Vroom , because it looses time by trying useless combinations :( but these "exploration" operators might be really useful in more realistic instances with timewindows etc.

Reply:
both or-tools and vroom (and MIP as well for that matter) has troubles when there is no TWs (or when the TWs are "permissively" large).

If 87% of the services have TWs are longer than 4 hours (almost half of which even longer than 6.5 hours...) yeah, that is almost like no TW. it might even be harder since solver needs to check the TW constraints for violation but they don't really decrease the size of the search space so it might be even worse.